widgets/nuxeo-directory-checkbox.html

```diff
@@ -0,0 +1,138 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../nuxeo-elements/nuxeo-operation.html">
+<link rel="import" href="../../iron-selector/iron-selector.html">
+<link rel="import" href="../../paper-checkbox/paper-checkbox.html">
+<link rel="import" href="../../iron-validatable-behavior/iron-validatable-behavior.html">
+<link rel="import" href="../../iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="nuxeo-directory-widget-behavior.html">
+
+<!--
+An element for selecting one or more directory entries with checkboxes.
+
+    <nuxeo-directory-checkbox directory-name="country" value="{{value}}"></nuxeo-directory-checkbox>
+
+@group Nuxeo Widgets
+@element nuxeo-directory-checkbox
+@demo demo/widgets/index.html
+@homepage http://www.nuxeo.org
+-->
+<dom-module id="nuxeo-directory-checkbox">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment">
+      :host {
+        display: block;
+        position: relative;
+        padding-bottom: 8px;
+      }
+
+      :host([hidden]) {
+        display: none;
+      }
+
+      :host([invalid]) .label,
+      .error {
+        color: var(--paper-input-container-invalid-color, red);
+      }
+
+      :host([invalid]) .error {
+        opacity: 1;
+        font-size: .923rem;
+      }
+
+      .label {
+         @apply --nuxeo-label;
+      }
+
+      .label[required]::after {
+        display: block;
+        content: '*';
+        position: absolute;
+        right: 0;
+        top: 0;
+        color: var(--paper-input-container-invalid-color, red);
+        font-size: 1.2em;
+      }
+
+      paper-checkbox {
+        margin-top: 10px
+      }
+
+    </style>
+
+    <nuxeo-operation id="op" op="Directory.SuggestEntries">
+    </nuxeo-operation>
+
+    <label class="label" hidden$="[[!label]]" required$="[[required]]">[[label]]</label>
+
+    <iron-selector attr-for-selected="name" multi selected-attribute="checked"
```

Comment:
Seems weird to use an `<iron-selector>` and not binding to its `selected-values`

Reply:
indeed and that's why tests were failing on other browsers ;)